packages/pwa-kit-create-app/scripts/create-mobify-app.js

```diff
@@ -737,6 +737,10 @@ Examples:
                 (x) => `"${x}"`
             ).join(', ')})`
         )
+        .option(
+            '--templateVersion <version>',
+            `The version of the template to be generated. This defaults to 'latest'. NOTE: This option only applies to templates being downloaded from NPM.`
+        )
```

Comment:
Let's add the default value `latest` here instead, as a 3rd argument to this `option`.